cross-doc-explainer.md

```diff
@@ -0,0 +1,114 @@
+
+Both documents can interrupt the transition or customize it using JavaScript in different phases.
+
+## Same-origin
+
+Cross-document view transitions are only enabled for same-origin navigations without a
+[cross-origin redirect](https://html.spec.whatwg.org/#unloading-documents:was-created-via-cross-origin-redirects).
+In the future we could examine exposing them in some way to same-site or cross-origin navigations.
+
+## Opt-in
+
+To enable cross-document transitions, both the old and new document need to add a meta tag. This tag
+indicates that the author wants to enable transitions for same-origin navigations to/from this
+Document.
+
+
+### Issues
+
+1. The meta tag can be used to opt-in to other navigation types going forward: same-document, same-site, etc.
+
+1. Using meta-tags prevents the declaration being controlled by media queries, which feels important for `prefers-reduced-motion`.
+
+## Lifecycle
+
+![Lifecycle chart for cross-document transitions](media/mpa-chart.svg)
+
+### Capturing the old state
+
+The old state is captured right before the old document is hidden and a new one is shown.
+In the HTML spec this is defined [here](https://html.spec.whatwg.org/#populating-a-session-history-entry:loading-a-document).
+This can either happen during normal navigations, when the new document is about to be created,
+in BFCache navigations, or when activating a prerendered document.
+
+Before creating the new document (or activating a cached/prerendered one), the UA would update the rendering and snapshot the old document, in the same manner a document is snapshotted for a same-document navigation.
+
+Note that currently there are no planned new events for the exit transition.
+The developer can use existing events like `navigate` or `click` to customize the
+exit transition at moments that should be late enough.
+
+#### Issues
+
+1. Should we enable skipping the transition on exit? This could be done on`pagehide`.
+
+### Capturing the new state
+
+The new state is captured at the first [rendering opportunity](https://html.spec.whatwg.org/#rendering-opportunity)
+of the new document. This allows the new document to use the
+[render-blocking mechanism](https://html.spec.whatwg.org/#render-blocking-mechanism) as a way to
+delay the transition.
+
+For most cases, opting in via the meta tag, styling with the pseudo-elements, and delaying the transition using the
+[render-blocking mechanism](https://html.spec.whatwg.org/#render-blocking-mechanism) should be sufficient. But there are certain cases where further customization is desired, for example:
+
+* Abort the transition if certain conditions apply.
+
+* Modify the pseudo-element style based on certain conditions, e.g. apply transition
+   names only to images that are already loaded.
+
+* Prepare the exit transition right before capture.
+
+There are several options as to how to enable this:
+
+1. Fire an event right before the new document capture. This would be either right
```

Comment:
Similar comment to above, "new document capture" sounds like a misnomer. Maybe "new document's shared element discovery" or something?

Reply:
See comment above, remaining consistent with the spec